index.html

```diff
@@ -79,61 +79,82 @@ <h5>I'm a software developer! I can help you build a product ,feature or website
+
+             <div class="frame2">
+                <ul class="card-frame">
+                    <li>CANOPY</li>
+                    <li class="no-count">Back End Dev</li>
+                    <li class="no-count">2015</li>
+                </ul>
+             </div>
+            </div>
+
+            <div class="img-popup">
+                <img class="popup-img">
+            </div>
+            
+            <div class="right-block">
+                <p class="popups-project-p-mobile">
+                    Lorem ipsum dolor sit amet consectetur adipisicing elit. Minus
+                    repellendus mollitia saepe maxime, vitae nostrum repellat eligendi
+                    amet voluptas modi repudiandae necessitatibus animi harum obcaecati
+                    at omnis laboriosam sapiente eum rem velit molestias,
+                  </p>
+                <ul class="card-tags2">
+                    <li>HTML</li>
+                    <li>css</li>
+                    <li>javascript</li>
+                </ul>
+                
+                <ul class="card-tags1">
+                    <li>Github</li>
+                    <li>Ruby</li>
+                    <li>Frontend</li>
+                </ul>
+                <div class="action">
+                    <button class="seeSource see"><p>See Source</p>
+                        <div class="see2"></div>
+                    </button>
+                    <hr class="divider">
+                    <button class="seeLive see"><p>See Live</p>
+                        <div class="see1"></div>
+                    </button>
+                </div>
+                <hr class="divider2">
+            </div>
+
+            <div class="left-block">
+                <p class="popups-project-p-desktop">Lorem Ipsum is simply dummy text of the printing and typesetting industry.
+                    Lorem Ipsum has been the industry's standard dummy text ever since the 1500s,
+                    when an unknown printer took a galley of type and scrambled it 1960s with
+                    the releaLorem Ipsum is simply dummy text of the printing and typesetting industry.
+                    Lorem Ipsum has been the industry's standard dummy text ever since the 1500s,
+                    when an unknown printer took a galley of type and scrambled it 1960s with the
+                    releorem Ipsum is simply dummy text of the printing and typesetting industry.
+                    Lorem Ipsum han printer took a galley of type and scrambled it 1960s with the
+                    releawn printer took a galley of type and scrambled it 1960s with the releaLorem
+                    Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum
+                    has been the industrys standard dummy text ever since the 1500s, when an unknown
+                    printer took a galley of type and scrambled it 1960s with the relea</p>
+
+
+            </div>
+        </div>
```

Comment:
- [ ] In your html file should be an empty div that will complete dynamically 👉🏽 `<div id="popup"></div>` I can see you haven't completed all the changes required from the last code reviewer, please 🙏🏽 fix it.

> Please make sure to remove the entire work section card from the main page, Work section cards should be created dynamically when the page loads with the information you stored in the javascript array as requested in the project requirements.